src/pages/my_page/MyPagePage.tsx

```diff
@@ -53,6 +55,8 @@ const MyPagePage = () => {
   const user = useSelector((store: RootState) => store.user);
   const isAuthenticated = !!user;
   const isCreator = !!user?.customerId && (user.isCreator || user.isCurator);
+  const personId = useSelector((store: RootState) => store.user?.cristinId) ?? '';
```

Comment:
user har vi allerede hentet over her

```suggestion
  const personId = user?.cristinId ?? '';
```